runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/PaymentTracker.java

```diff
@@ -90,6 +100,48 @@ private static String configKey(FarmingPatch fp)
 		return fp.configKey() + "." + TimeTrackingConfig.PROTECTED;
 	}
 
+	@Subscribe
+	public void onMenuOptionClicked(MenuOptionClicked menuOptionClicked)
+	{
+		switch (menuOptionClicked.getMenuAction())
+		{
+			case NPC_THIRD_OPTION:
+				chosenPatch = 1;
```

Comment:
This should definitely either check the option is Pay (North/South/whatever) here, or maybe check the NPC is a farmer (or both?). This will get set any time you use op3 on any npc.